src/adapter/graphql/src/queries/accounts/account_flows.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use opendatafabric as odf;
+
+use super::AccountFlowRuns;
+use crate::prelude::*;
+
+///////////////////////////////////////////////////////////////////////////////
+
+pub struct AccountFlows {
+    account_id: odf::AccountName,
```

Comment:
Everywhere - do not confuse yourself. ID is ID, name is name.

Reply:
Changed